contentcuration/contentcuration/frontend/shared/data/changes.js

```diff
@@ -383,18 +383,19 @@ export class DeletedChange extends Change {
 }
 
 export class MovedChange extends Change {
-  constructor({ target, position, parent, oldParent, ...fields }) {
+  constructor({ oldObj, target, position, parent, ...fields }) {
     fields.type = CHANGE_TYPES.MOVED;
     super(fields);
     if (this.table !== TABLE_NAMES.CONTENTNODE) {
       throw TypeError(
         `${this.changeType} is only supported by ${TABLE_NAMES.CONTENTNODE} table but ${this.table} was passed instead`
       );
     }
+    this.setAndValidateObj(oldObj, 'oldObj', omitIgnoredSubFields);
     this.setAndValidateIsDefined(target, 'target');
     this.setAndValidateLookup(position, 'position', RELATIVE_TREE_POSITIONS_LOOKUP);
     this.setAndValidateIsDefined(parent, 'parent');
-    this.setAndValidateIsDefined(oldParent, 'oldParent');
+    this.setAndValidateIsDefined(oldObj.parent, 'oldParent');
```

Comment:
We really don't need to `setAndValidateIsDefined` both `oldObj` and `oldParent` since, as your change here suggests, the `oldParent` is accessible via `oldObj`. So I think we should remove `oldParent` and update usages of `oldParent` to use `oldObj.parent` instead.